portality/crosswalks/article_crossref_xml.py

```diff
@@ -171,13 +172,44 @@ def crosswalk_article(self, record, journal):
         md = journal.find("x:journal_metadata", NS)
         if md is not None:
             issns = md.findall("x:issn", NS)
-            if issns is not None:
-                for issn in issns:
-                    if issn.attrib["media_type"] is None or issn.attrib["media_type"] == 'print':
-                        bibjson.add_identifier(bibjson.P_ISSN, issn.text.upper())
-                    elif issn.attrib["media_type"] == 'electronic':
-                        bibjson.add_identifier(bibjson.E_ISSN, issn.text.upper())
 
+            # if more than 2 issns raise the exception
+            if len(issns) > 2:
+                raise CrosswalkException(message="Too many ISSNs. Only 2 ISSNs are allowed")
```

Comment:
Let's put the text itself in the Messages file, and then I think this is good to release